recipes/cellprofiler-core/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "cellprofiler-core" %}
+{% set version = "4.0.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/cellprofiler-core-{{ version }}.tar.gz
+  sha256: bf169eeae8f6222355acf27ed85e9d9f21d40d4e7b715eda451200b728995d1a
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6,<4.0
+  run:
+    - boto3 >=1.12.28
+    - centrosome ==1.2.0
+    - docutils ==0.15.2
+    - h5py >=2.10.0
+    - matplotlib-base >=3.1.3
+    - numpy >=1.18.2
+    - prokaryote ==2.4.2
+    - psutil >=5.7.0
+    - python >=3.6,<4.0
+    - python-bioformats ==4.0.0
+    - python-javabridge ==4.0.0
+    - pyzmq ==18.0.1
+    - scikit-image >=0.16.2
+    - scipy >=1.4.1
+
+test:
+  imports:
+    - cellprofiler_core
+    - cellprofiler_core.analysis
+  commands:
+    - pip check
```

Comment:
why do you test pip here?

Reply:
I generated this recipe using the `grayskull` tool which adds these lines. I'm not too sure of its relevance here, so happy to remove.